projects/rocfft/shared/fft_params.h

```diff
@@ -366,7 +430,8 @@ inline void set_input(std::vector<hostbuf>&      input,
                       const Tint1                field_lower,
                       const size_t               field_lower_batch,
                       const Tint1                field_contig_stride,
-                      const size_t               field_contig_dist)
+                      const size_t               field_contig_dist,
+                      const bool                 use_gpu_symmetrizer)
```

Comment:
```suggestion
                      const bool                 use_gpu_symmetrizer = false)
```
to keep its default behavior unchanged (unless it's decided to leave this unchanged in the end).